src/components/cylc/tree/index.js

```diff
@@ -18,6 +18,47 @@
 // eslint-disable-next-line no-unused-vars
 import CylcTree from '@/components/cylc/tree/cylc-tree'
 
+/**
+ * These are the properties that are displayed on each Job detail, the leaf node of the tree.
+ * @type {({property: string, title: string}|{property: string, title: string}|{property: string, title: string}|{property: string, title: string}|{property: string, title: string})[]}
+ */
+const JOB_DETAIL_NODE_PROPERTIES = [
+  {
+    title: 'host id',
+    property: 'host'
+  },
+  {
+    title: 'job id',
+    property: 'batchSysJobId'
+  },
+  {
+    title: 'batch sys',
+    property: 'batchSysName'
+  },
+  {
+    title: 'submit time',
+    property: 'submittedTime'
+  },
+  {
+    title: 'start time',
+    property: 'startedTime'
+  },
+  {
+    title: 'finish time',
+    property: 'finishedTime'
+  },
+  {
+    title: 'latest message',
+    property: 'latestMessage'
+  }
+]
+
+/**
+ * The height of each tree item. In the infinite tree, it is used as value for the `.size` property.
```

Comment:
>Another quick question, what happens if the user changes the font-size?

Huh. Hadn't thought about that. I did a quick test, first with the workflow held. And everything worked fine.

The I released it, and resized window so that there were two scroll bars (one for the window area, one for the tree view). And started scrolling both, while increasing/decreasing the font size. Apparently it worked OK.

We have an issue for accessibility, so I think it would still be a good idea to leave a comment there saying that the infinite tree calculates heights with pixels (for now, will check if we can use `em` instead) and that we need to make sure changing the font size doesn't cause unexpected issues in the tree view.

WDYT?

Reply:
>A potential enhancement for a later PR, could we write a functional test which checks that the height is in-fact 32 pixels so that any theme changes don't accidentally break the infinite tree.

+1!

And this would be a good issue for any new contributor.